src/rdf4cpp/rdf/datatypes/xsd/integers/non_negative/PositiveInteger.cpp

```diff
@@ -48,18 +48,21 @@ nonstd::expected<capabilities::Default<xsd_positive_integer>::cpp_type, DynamicE
 }
 
 template<>
-std::optional<uint64_t> capabilities::Inlineable<xsd_positive_integer>::try_into_inlined(cpp_type const &value) noexcept {
+std::optional<storage::node::identifier::LiteralID> capabilities::Inlineable<xsd_positive_integer>::try_into_inlined(cpp_type const &value) noexcept {
     auto const to_pack_value = value - 1;
     if (to_pack_value >= (uint64_t{1} << storage::node::identifier::LiteralID::width)) {
         return std::nullopt;
     }
 
-    return util::try_pack_integral<uint64_t, storage::node::identifier::LiteralID::width>(static_cast<uint64_t>(to_pack_value));
+    auto p = util::try_pack_integral<uint64_t, storage::node::identifier::LiteralID::width>(static_cast<uint64_t>(to_pack_value));
```

Comment:
- [ ] Task for @Clueliss check if that can be done directly without optional fiddling